app/src/main/java/com/cortado/mafkir/di/ViewModelFactoryModule.kt

```diff
@@ -0,0 +1,15 @@
+package com.cortado.mafkir.di
+
+import androidx.lifecycle.ViewModelProvider
+import com.cortado.mafkir.util.ViewModelProviderFactory
+import dagger.Binds
+import dagger.Module
+
+
+@Module
+abstract class ViewModelFactoryModule {
+
+    // Method #2
+    @Binds
+    abstract fun bindViewModelFactory(viewModelProvideFactory: ViewModelProviderFactory): ViewModelProvider.Factory
```

Comment:
`viewModelProvideFactory`: did you mean `viewModelProviderFactory`?

Reply:
d